server/api/migrations/versions/a1973a4e89df_added_foo_to_functionspec.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Added Foo to FunctionSpec
+
+Revision ID: a1973a4e89df
+Revises: fcf2ea01f99a
+Create Date: 2024-09-03 17:29:02.498311
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "a1973a4e89df"
+down_revision = "fcf2ea01f99a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
shouldn't this migration add the relationship to the functions table as well?

Reply:
WDYM?
The function.id becomes a foreign key on the function_spec_foos table, the relationship is managed by SQLAlchemy when it is defined on the ORM class.